modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -944,6 +944,7 @@ add-multiple-numeric-fields-together-and-return-a-single-number-that-can-be-used
 add-new=Add New
 add-new-address=Add New Address
 add-new-api-application=Add New API Application
+add-new-attribute-row=Add new attribute row
```

Comment:
Since this does not end in a period, it should follow title case. `Add New Attribute`. Do we need to call it a row?